### PR TITLE
Trend pytmv1

### PR DESCRIPTION
**Pull request by shaqnawe**
## Status
Ready

## Related Content Pull Request
Trend micro vision one #35385
https://github.com/demisto/content/pull/35385

## Related Issues
Related: link to the issue

## Description
Updated pytmv1 package to 0.8.7.

### CODE REVIEW

**Comment by JudahSchwartz** [comment]
from your build
`[ERROR] Pytmv1: Version mismatch or version is invalid format. The Pipfile version 3.10 does not match to the base version 3.11.9.Change it to the X.Y format.`
we recently updated the base to python 3.11
Please update the pipfile to use python 3.11 and run pipenv lock in the directory of the pipfile

**Comment by shaqnawe** [comment]
@JudahSchwartz python version has been updated to 3.11. Please advise regarding verify script error.

**Comment by t0mz06** [comment]
Hello @JudahSchwartz, please could you help check asap this PR? We have a customer waiting on the fix, it would be greatly appreciated.

Thank you

**Comment by t0mz06** [comment]
Hello @thefrieddan1 , we are not able to see why the verify script is failing, could you please help? I tried to import on python 3.11 and it worked fine, not sure what is happening

**Review by thefrieddan1 — APPROVED** [review]

**Comment by thefrieddan1** [comment]
@shaqnawe @t0mz06  Im currently running the ci pipeline and will merge once it finishes successfuly.

**Comment by thefrieddan1** [comment]
The following CVE-2024-6345 was detected and prevents the merge.
Please use pypa/setuptools version >= 70.0

**Comment by shaqnawe** [comment]
> The following [CVE-2024-6345](https://github.com/advisories/GHSA-cx63-2mw6-8hw5) was detected and prevents the merge. Please use pypa/setuptools version >= 70.0

@thefrieddan1 pytmv1 doesn't use setuptools, how can I update this? I tried adding it to pipfile but it still doesn't reflect. Please advise.

File: docker/pytmv1/Pipfile [hunk 1]
```diff
@@ -5,8 +5,9 @@ name = "pypi"
 
 [packages]
 pytmv1 = "*"
+setuptools = "70.3.0"
```
Comment (JudahSchwartz) [inline docker/pytmv1/Pipfile:8]: this can be removed. @adi88d is working on this in the base image

Comment (shaqnawe) [reply docker/pytmv1/Pipfile:8]: thank you, will remove this.

**Pull request merged**